README.md

```diff
@@ -2,209 +2,185 @@
 
 # PEGParser
 
-PEGParser is a PEG Parser for Julia with Packrat capabilties. PEGParser was inspired by pyparsing, parsimonious, boost::spirit, as well as several others.
+PEGParser is a parsing library for Parsing Expression Grammars (PEG) in Julia. It was inspired by pyparsing, parsimonious, boost::spirit, as well as several others. The original design was set up by Abe Schneider in 2014. As of January 2017 Henry Schurkus has reworked major parts of the library design. With the redesign also came a change of the API for easier and less error prone use. Below we describe the new design which takes grammar declarations in the form of (multiline) strings. The previous design relied heavily on the specific parsing logic of the julia language and should therefore be considered deprecated.
+
+# Super Quick Tutorial For The Very Busy
+A parser takes a string and a grammar specification to turn the former into a computable structure. PEGParser does this by first parsing (`parse(grammar,string)`) the string into an Abstract Syntax Tree (AST) and then transforming this AST into the required structure (`transform(function,AST)`).
+
 ## Defining a grammar
 
-To define a grammar you can write:
+A grammar can be defined as:
 
 ```julia
-@grammar <name> begin
+Grammar("""
   rule1 = ...
```

Comment:
Sorry for the late reply -- no need to put my name on the library since it looks like it's taking a different direction. Instead, if you like, you can just put based on PEGParser.